projects/core/src/product/model/product-scope.ts

```diff
@@ -3,4 +3,5 @@ export enum ProductScope {
   DETAILS = 'details',
   ATTRIBUTES = 'attributes',
   VARIANTS = 'variants',
+  BULK_PRICES = 'bulkPrices',
```

Comment:
Is is possible to move this one also to the library and use type augmentation?

Reply:
Augmented enum feature lib.